docs/src/prototypes/chatMessages/ChatMessageWithPopover/ChatWithPopover.tsx

```diff
@@ -1,124 +1,147 @@
-                author="Jane Doe"
-                content={{
-                  content: (
-                    <div>
-                      <a href="/">Link</a> Hover me to see the actions <a href="/">Some Link</a>
-                    </div>
-                  ),
-                }}
-                timestamp="Yesterday, 10:15 PM"
-              />
-            ),
+      }}
+    >
+      <Chat
+        items={[
+          {
+            key: 'a',
+            message: {
+              content: (
+                <TeamsChatMessage
+                  author="Jane Doe"
+                  content={{
+                    content: (
+                      <div>
+                        <a href="/">Link</a> Hover me to see the actions <a href="/">Some Link</a>
+                      </div>
+                    ),
+                  }}
+                  reactionGroup={{
+                    items: reactionsWithPopup,
+                  }}
+                  timestamp="Yesterday, 10:15 PM"
+                />
+              ),
+            },
+            gutter: { content: <Avatar {...janeAvatar} /> },
           },
-          gutter: { content: <Avatar {...janeAvatar} /> },
-        },
-        {
-          key: 'c',
-          message: {
-            content: (
-              <Chat.Message
-                actionMenu={<Popover />}
-                author="Jane Doe"
-                content={{
-                  content: (
-                    <div>
-                      <a href="/">Link</a> Hover me to see the actions <a href="/">Some Link</a>
-                    </div>
-                  ),
-                }}
-                timestamp="Yesterday, 10:15 PM"
-              />
-            ),
+          {
+            key: 'b',
+            message: {
+              content: (
+                <TeamsChatMessage
+                  author="Jane Doe"
+                  content={{
+                    content: (
+                      <div>
+                        <a href="/">Link</a> Hover me to see the actions <a href="/">Some Link</a>
+                      </div>
+                    ),
+                  }}
+                  reactionGroup={{
+                    items: reactionsWithPopup,
+                  }}
+                  timestamp="Yesterday, 10:15 PM"
+                />
+              ),
+            },
+            gutter: { content: <Avatar {...janeAvatar} /> },
           },
-          gutter: { content: <Avatar {...janeAvatar} /> },
-        },
-      ]}
+        ]}
+      />
+    </Provider>
+  )
+}
+
+const TeamsChatMessage: React.FC<ChatMessageProps> = (props: ChatMessageProps) => {
+  const [open, setOpen] = React.useState(false)
+  const [fixedMode, setFixedMode] = React.useState(false)
+
+  return (
+    <Chat.Message
+      {...props}
+      actionMenu={<Popover setFixedMode={setFixedMode} setOpen={setOpen} />}
+      onMouseEnter={() => setOpen(true)}
+      onMouseLeave={() => !fixedMode && setOpen(false)}
```

Comment:
On mouse leave we will close the action menu, unless it is in fixed mode (the submenu is opened).